src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -54,14 +54,14 @@ public void start(Stage primaryStage) {
         this.primaryStage = primaryStage;
         logger.info("Starting UI...");
 
-        //Set the application icon.
+        //Set the application icon
         primaryStage.getIcons().add(getImage(ICON_APPLICATION));
 
         try {
             mainWindow = new MainWindow(primaryStage, config, prefs, logic, security);
             mainWindow.show(); //This should be called before creating other UI parts
             //Gets rid of the login window when testing
```

Comment:
This whole portion is just to make it easier when developing. Logically it should just show the CLI immediately, which will be done when the product is to be released.